daemon/src/db.rs

```diff
@@ -380,7 +982,11 @@ pub trait CfdAggregate: Clone + Send + Sync + 'static {
     fn version(&self) -> u32;
 }
 
-async fn load_cfd_row(conn: &mut PoolConnection<Sqlite>, id: OrderId) -> Result<Cfd> {
+pub trait ClosedCfdAggregate: CfdAggregate {
```

Comment:
Interesting that these traits live in the database. I feel that should probably be driven by the model. But we don't have to solve this as part of this PR.

Reply:
I did not think hard about where to put this trait because I followed what was already there i.e. the `CfdAggregate` trait.

I think they belong here because their purpose is to construct aggregates from _database_ types. Which is why they take `db::Cfd` and `db::ClosedCfd` as arguments.

Reply:
If they belong to the `db` I find it a bit odd that we re-use them in other parts of the code. Not super important though. We can drop this discussion for now or discuss somewhere else.